resources/megameklab/resources/Splash.properties

```diff
@@ -0,0 +1,15 @@
+btnLoadUnit.text=Load Existing Unit
+btnNewMek.text=New Mek
+btnNewVee.text=New Combat Vehicle
+btnNewSupportVee.text=New Support Vehicle
+btnNewProto.text=New Protomech
+btnNewBA.text=New Battle Armor
+btnNewPbi.text=New Infantry Unit
+btnNewAero.text=New Aero Fighter
+btnNewDropper.text=New Dropship/Small Craft
+btnNewLargeCraft.text=New Large Craft
```

Comment:
I think capital ship is more natural than "advanced aerospace" units although its weird to  all a space station a capitol ship.

Reply:
Might seem like an odd idea, but a button for Space Stations that jumps to the same starting point, but changes the drop to space station.  Or even button for JS,WS,SS that does that?

Reply:
The problem is that we are pretty maxed out on buttons already and any space we have I would rather reserve for other future things that @neoancient will code for us 😉

Reply:
I don't know if there's a really good solution for it. I raise it because being nitpicky can be asset when doing this sort of thing as long as you're not being a jerk about it (and I try not to be). If it goes out like this and people's feathers get ruffled it's no big deal to change it later. As far as what may get added in the future, I think we're down to mobile structures and handheld weapons, and HHWs may actually work better as part of mech construction.

Reply:
I changed it to "Capital-Scale Craft" which I think is accurate since these are all units that use capital scale armor.